packages/react-primitives/src/components/delayed/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect, type ReactNode } from "react";
+
+type Props = {
+  delayMs: number;
+  children: ReactNode;
+  fallback?: ReactNode;
+};
+
+export function Delayed({ delayMs, children, fallback }: Props) {
```

Comment:
Some TS docs would be great + maybe have an optional `className` prop? I can imagine a use case where we wanted to animate the transition